hoover/contrib/oauth2/views.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import redirect
+from django.contrib.auth.models import User
+from django.contrib.auth import login, logout
+from django.conf import settings
+import requests
+
+LIQUID_URL = settings.HOOVER_OAUTH_LIQUID_URL
+LIQUID_CLIENT_ID = settings.HOOVER_OAUTH_LIQUID_CLIENT_ID
+LIQUID_CLIENT_SECRET = settings.HOOVER_OAUTH_LIQUID_CLIENT_SECRET
+
+def oauth2_login(request):
+    authorize_url = LIQUID_URL + '/o/authorize/'
+    return redirect(
+        '{}?response_type=code&client_id={}'
+        .format(authorize_url, LIQUID_CLIENT_ID)
+    )
+
+def oauth2_exchange(request):
+    token_url = LIQUID_URL + '/o/token/'
+    redirect_uri = request.build_absolute_uri('/accounts/oauth2-exchange/')
+    code = request.GET.get('code')
+    token_resp = requests.post(
+        token_url,
+        data={
+            'redirect_uri': redirect_uri,
+            'grant_type': 'authorization_code',
+            'code': code,
+        },
+        auth=(LIQUID_CLIENT_ID, LIQUID_CLIENT_SECRET),
+    )
+    assert token_resp.status_code == 200
```

Comment:
Could we have a more comprehensive error message show up in the logs? 

The assert doesn't really help future debugging efforts.
Maybe even log the response status code, so one could investigate further.